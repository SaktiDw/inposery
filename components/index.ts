export { default as Navigation } from "./Nav/Navigation";
export { default as Sidebar } from "./Nav/Sidebar";
export { default as SidebarItem } from "./Nav/SidebarItem";
export { default as Dashboard } from "./Layouts/Dashboard";
export { default as StoreDashboard } from "./Layouts/StoreDashboard";
export { default as DashboardCard } from "./Cards/DashboardCard";
export { default as StoreCard } from "./Cards/StoreCard";
export { default as ProductCard } from "./Cards/ProductCard";
export { default as OrderCard } from "./Cards/OrderCard";
export { default as Table } from "./Tables/Table";
export { default as Pagination } from "./Tables/Pagination";

export { default as Modal } from "./Modals/Modal";
export { default as StoreForm } from "./Forms/StoreForm";
export { default as ProductForm } from "./Forms/ProductForm";
export { default as CashierForm } from "./Forms/CashierForm";
export { default as TransactionForm } from "./Forms/TransactionForm";
export { default as Input } from "./Inputs/Input";
export { default as SearchInput } from "./Inputs/SearchInput";
export { default as PerPageSelect } from "./Inputs/PerPageSelect";
export { default as Dropzone } from "./Inputs/Dropzone";

export { default as PriceFormater } from "./FormatText/PriceFormater";
export { default as Receipts } from "./Print/Receipts";

export { default as SalesChart } from "./Chart/SalesChart";
export { default as ComparationChart } from "./Chart/ComparationChart";
