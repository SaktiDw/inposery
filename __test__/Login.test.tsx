import Login from "@/pages/login";
import { describe, expect, it, test, vitest } from "vitest";
import { render, screen, waitFor } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import { StoreForm } from "@/components";
import Stores from "@/pages/store";

describe("Login Page Testing", () => {
  it("Should be render Login Page", async () => {
    const { container } = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
