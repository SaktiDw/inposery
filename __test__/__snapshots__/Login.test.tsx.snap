// Vitest Snapshot v1

exports[`Login Page Testing > Should be render Login Page 1`] = `
<div>
  <div
    class="w-full min-h-screen bg-slate-100 text-slate-800 dark:bg-slate-800 dark:text-white flex items-center justify-center"
  >
    <form
      action="#"
      class="flex flex-col gap-4 w-1/2"
    >
      <div
        class="text-3xl"
      >
        Login
      </div>
      <div
        class="flex flex-col gap-2"
      >
        <label
          class="font-semibold leading-tight text-slate-800 dark:text-white"
          for="email"
        >
          E-mail
        </label>
        <input
          class="py-2 px-4 rounded-lg w-full text-slate-800 dark:text-white bg-transparent ring-2 shadow-md ring-lime-500"
          id="email"
          name="email"
          placeholder="e.g example@gmail.com"
          required=""
          type="email"
          value=""
        />
      </div>
      <div
        class="flex flex-col gap-2"
      >
        <label
          class="font-semibold leading-tight text-slate-800 dark:text-white"
          for="password"
        >
          Password
        </label>
        <input
          class="py-2 px-4 rounded-lg w-full text-slate-800 dark:text-white bg-transparent ring-2 shadow-md ring-lime-500"
          id="password"
          name="password"
          placeholder="password"
          required=""
          type="password"
          value=""
        />
      </div>
      <button
        class="w-full  text-white  bg-green-500 ring-2 ring-green-500 disabled:ring-green-800 disabled:bg-green-800 disabled:cursor-not-allowed disabled:shadow font-semibold shadow-xl py-2 px-4 rounded-lg "
        type="submit"
      >
        Submit
      </button>
      <span
        class="text-slate-500"
      >
        Dont have an account?
         
        <a
          href="/register"
        >
          register here
        </a>
      </span>
    </form>
  </div>
</div>
`;
